Cache_Redis.php

```diff
@@ -331,7 +331,7 @@ public function counter_get( $key ) {
 	}
 
 	private function _get_accessor( $key ) {
-		if ( count( $this->_servers ) <= 1 )
+		if ( count( $this->_servers ) <= 1 || W3TC_REDIS_CLUSTER )
```

Comment:
I better condition would be:
`if ( count( $this->_servers ) <= 1 || ( defined( 'W3TC_REDIS_CLUSTER' ) && W3TC_REDIS_CLUSTER ) ) {`